ebrains_drive/file.py

```diff
@@ -26,4 +28,31 @@ def get_file_by_url(self, file_url):
         repo_obj = self.client.repos.get_repo(repo_id)
         file_obj = repo_obj.get_file(file_path)
         return file_obj
-        
\ No newline at end of file
+
+    def get_file_by_local_path(self, local_path):
+        """
+        Get the file or directory object corresponding to `local_path`
+        when the Drive is mounted in the EBRAINS Lab.
+        """
+        home_dir = "/mnt/user/drive/My Libraries/My Library/"
+        group_dir = "/mnt/user/drive/Shared with groups/"
+        shared_dir = "/mnt/user/shared/"
+        if local_path.startswith(home_dir):
+            repo_obj = self.client.repos.get_default_repo()
+            relative_path = os.path.relpath(local_path, home_dir)
+        elif local_path.startswith(group_dir):
+            collab_name = local_path.split("/")[5]
```

Comment:
I will replace it by a simple `local_path.replace(group_dir,"")` so we won't have issue if the amount of `/` is different if there is a `/` in the collab title.